lib/surface/components/link.ex

```diff
@@ -57,7 +57,7 @@ defmodule Surface.Components.Link do
     children = ~H"<slot>{{ @label }}</slot>"
 
     ~H"""
-    {{ link [to: @to, class: @class] ++ @opts ++ event_to_opts(@click, :phx_click), do: children }}
+    {{ link [to: @to] ++ prop_to_opts(__MODULE__, :class, @class) ++ @opts ++ event_to_opts(@click, :phx_click), do: children }}
```

Comment:
You'll have to change this to pass the env instead of the module:

```
    {{ link [to: @to] ++ prop_to_opts(:class, @class, __ENV__) ++ @opts ++ event_to_opts(@click, :phx_click), do: children }}
```